setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     easy-thumbnails==2.2
     edx-gea==0.2.0
     glowbl-xblock==0.2.0
+    oauth2client==3.0.0
```

Comment:
Also add a comment here to explain why it must be installed before `google-api-python-client` (and thus why you are not respecting alphabetical order :stuck_out_tongue:).